packages/shared/components/organisms/AccountActivity.svelte

```diff
@@ -21,6 +27,55 @@
         })
     }
 
+    async function _syncAccount() {
+        $isSyncing = true
+        try {
+            await get(selectedAccount).sync()
```

Comment:
Are we doing anything after sync? Don't we need to load any new data into our stores?